cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go

```diff
@@ -96,8 +95,9 @@ var (
 // GceInstance extends cloudprovider.Instance with GCE specific numeric id.
 type GceInstance struct {
 	cloudprovider.Instance
-	NumericId uint64
-	Igm       GceRef
+	NumericId        uint64
+	Igm              GceRef
+	InstanceTemplate string
```

Comment:
```suggestion
	InstanceTemplateName string
```
I think it's worth it to avoid ambiguity.

Reply:
Done, thank you for the review!